modules/openapi-generator/src/main/resources/python/api_client.handlebars

```diff
@@ -293,7 +293,7 @@ class StyleFormSerializer(ParameterSerializerBase):
         prefix_separator_iterator: typing.Optional[PrefixSeparatorIterator] = None
     ) -> str:
         if prefix_separator_iterator is None:
-            prefix_separator_iterator = PrefixSeparatorIterator('?', '&')
```

Comment:
In the caller `__serialize_application_x_www_form_data`, please set percent_encode to True:
```
    def __serialize_application_x_www_form_data(
        self, in_data: typing.Any
    ) -> SerializedRequestBody:
        """
        POST submission of form data in body
        """
        if not isinstance(in_data, frozendict.frozendict):
            raise ValueError(
                f'Unable to serialize {in_data} to application/x-www-form-urlencoded because it is not a dict of data')
        cast_in_data = self.__json_encoder.default(in_data)
        value = self._serialize_form(cast_in_data, name='', explode=True, percent_encode=True)
        return dict(body=value)
```
Which is required per https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/POST

Reply:
Done. I added the related change to the unit test. I also did a manual test with the oauth endpoint on the production Pinterest API to double-check that it accepts percent-encoded data from the generated openapi client.

Reply:
Thanks!